### PR TITLE
DOC: 1.9.1 relnotes update

### PR DESCRIPTION
**Pull request by tylerjereddy**
* update the release notes for SciPy `1.9.1`
after a few more backports

[skip azp] [skip actions]

### CODE REVIEW

File: doc/release/1.9.1-notes.rst [hunk 1]
```diff
@@ -36,6 +37,7 @@ Issues closed for 1.9.1
 Pull requests for 1.9.1
 -----------------------
 
+* `#16709 <https://github.com/scipy/scipy/pull/16709>`__: BLD: make the way we count commits for version numbering more...
 * `#16736 <https://github.com/scipy/scipy/pull/16736>`__: REL: prep for SciPy 1.9.1
 * `#16749 <https://github.com/scipy/scipy/pull/16749>`__: BLD: install missing \`.pxd\` files, and update TODOs/FIXMEs...
 * `#16750 <https://github.com/scipy/scipy/pull/16750>`__: BLD: make OpenBLAS detection work with CMake
```
Comment (WarrenWeckesser) [inline doc/release/1.9.1-notes.rst:43]:
```suggestion
* `#16750 <https://github.com/scipy/scipy/pull/16750>`__: BLD: make OpenBLAS detection work with CMake
* `#16755 <https://github.com/scipy/scipy/pull/16755>`__: TST: sparse.linalg: Loosen tolerance for the lobpcg test 'test_tolerance_float32'
```

**Comment by tylerjereddy** [comment]
Not sure why Azure is running again, both commits have `[skip azp] [skip actions]`, maybe b/c of the way suggested commits work..

I guess I'll wait for this to return green then, but the main thing I wanted to see was the docs build success, which already happened.

**Comment by tylerjereddy** [comment]
CI was green including doc build, merged.

**Comment by tylerjereddy** [comment]
wheel builds started

**Pull request merged**